conda/recipes/libcuspatial/meta.yaml

```diff
@@ -18,6 +18,7 @@ build:
     - CC
     - CXX
     - CUDAHOSTCXX
+    - CUDAToolkit_ROOT
```

Comment:
Is this necessary? How are we handling it for libcudf?

Reply:
I added it to see if it got around the CMake 3.18.x issue when we symlink `/usr/local/bin/nvcc` to ccache, but I think we can remove it now.